inst/odin/lancelot.R

```diff
@@ -66,6 +77,38 @@ initial(cum_n_vaccinated[, ]) <- 0
 update(cum_n_vaccinated[, ]) <- cum_n_vaccinated[i, j] + n_vaccinated[i, j]
 dim(cum_n_vaccinated) <- c(n_groups, n_vacc_classes)
 
+
```

Comment:
again maybe adding more comments in code to say because we counted the people who skip classes together with the others in the outputted numbers of  people who have progressed to a given vaccine class, but we may want to monitor the skipping vs non skipping flow separately, we are also outputting the skipping only

Reply:
Added a comment about this